rest/servers/spring/src/main/java/com/connexta/ingest/rest/spring/IngestApi.java

```diff
@@ -72,30 +72,32 @@
   default ResponseEntity<Void> ingest(
       @ApiParam(
               value =
-                  "The minimal API version that a client using this API will accept responses from. ",
+                  "The minimal API version from which the client using this API will accept responses. ",
               required = true)
           @RequestHeader(value = "Accept-Version", required = true)
           String acceptVersion,
       @ApiParam(
               value =
-                  "The last modified time of the product to be ingested. ",
+                  "The last modified time of the dataset to be ingested. ",
```

Comment:
❓ This is the last modified time of a collection of files and metadata? Seems odd to me. Should it the last modified of the file?

Reply:
Good suggestion. I was thinking that the `metacard` can be updated even if the `file` hasn't been updated. Is this right? So, it would be the last modified timestamp of the `file` or `metadata`. Since the `dataset` only contains the `file` and `metadata`, does `The last modified time of the dataset` work?

Reply:
Yeah, technically each file and metadata can have its own last modified. Eventually maybe we need to pass it in on a per file and metadata basis?

For now, replication passes in what is on the metacard's last modified, so we should be able to get away with the saying it is the last modified of the metacard.

Reply:
I think replication passes in the last modified time of the product not of the metacard.

Reply:
Replication uses metacard.modified for the timestamp that it sends to Ion: https://github.com/connexta/replication/blob/773d1a04347f53e4638bf8074145d621b4f12f6d/adapters/ion-adapter/src/main/java/com/connexta/replication/adapters/ion/IonNodeAdapter.java#L127-L128. In DDF systems, the metacard is always modified when the product is modified.

Reply:
If the endpoint ingests a dataset, the last modified timestamp should refer to the dataset. The meaning of "dataset last modified" can be left to the endpoint consumer. If the consumer knows how CDR will use the timestamp to reject/accept requests, then the consumer can send the timestamp most appropriate their intentions.